examples/s2-parcel-example/src/App.js

```diff
@@ -236,6 +340,12 @@ function App() {
             The missing link.
           </Link>
           <Link href="/foo">Foo</Link>
+          <SegmentedControl styles={style({width: 384})}>
```

Comment:
<img width="438" alt="Screenshot 2024-10-24 at 10 51 33 AM" src="https://github.com/user-attachments/assets/192d6788-a9c1-4371-ae79-1bb4739cee4d">

It looks like segmented control is missing some styles for some reason. not sure what's going on here. seems to be happening in the parcel and webpack app

Reply:
or actually, it seems like all of them have `data-selected` which is why they look this way

Reply:
also we should probably add an aria label to the segmented control so that the warnings don't come up in the console

Reply:
> or actually, it seems like all of them have data-selected which is why they look this way

I think I need to use `id` instead of `value`. Not sure why I have that, maybe copied from the old PR before we changed.

> also we should probably add an aria label to the segmented control so that the warnings don't come up in the console

Wonder why we didn't hit this in storybook. Maybe the above will fix it? We shouldn't need an aria-label if there is just text children.

Reply:
We may want to delete the `value` prop to prevent this so it doesn't make it into the Radio

Reply:
i don't think `SegmentedControlItem` has a `value` prop so it should be fine. we do pass the `id` to the Radio underneath tho

> Wonder why we didn't hit this in storybook

the storybook examples have an aria-label on them so that's why there haven't been any warnings. i think we do need it because it uses RadioGroup under the hood. things might change with toolbar though

Reply:
> the storybook examples have an aria-label

ahh ok I was looking at the items